src/AltinnCore/Common/AltinnCore.Common.csproj

```diff
@@ -16,7 +16,7 @@
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.CodeAnalysis" Version="2.8.2" />
     <PackageReference Include="LibGit2Sharp" Version="0.26.0" />
-    <PackageReference Include="JWTCookieAuthentication" Version="0.8.1-alpha" />
+    <PackageReference Include="JWTCookieAuthentication" Version="0.8.7-alpha" />
```

Comment:
I guess this should now be update to 0.8.8?

Reply:
Only runtime is dependent on the higher version package. As we are anticipating more updated over the next days I figure we wait to update the rest of the projects untill we have a more stable nuget package. Only the logging has changed anyways